Kernel/Net/NetworkAdapter.h

```diff
@@ -70,13 +82,20 @@ class NetworkAdapter : public RefCounted<NetworkAdapter> {
     u32 packets_out() const { return m_packets_out; }
     u32 bytes_out() const { return m_bytes_out; }
 
+    RefPtr<PacketWithTimestamp> acquire_packet_buffer(size_t);
+    void release_packet_buffer(PacketWithTimestamp&);
+
+    size_t layer3_payload_offset() const;
```

Comment:
These should be `constexpr`?

Reply:
I've made `layer3_payload_offset()` and `ipv4_payload_offset()` constexpr but couldn't make those other two constexpr.